config/config.go

```diff
@@ -272,7 +272,6 @@ func WriteDefault(path string, conf *Config) {
 		ListenPort:     8080,
 		Secret:         "352d20ee67be67f6340b4c0605b044b7",
 		TemplatePath:   "templates",
-		TykJSPath:      "js/tyk.js",
```

Comment:
I assume you should rollback this config option?

Reply:
No - this sets the default. We no longer ship or have `js/tyk.js`, so this needs to go. If a user wants to use global JS code, they need to set the config option to whatever path they like.

Reply:
Right